### PR TITLE
wasm: clippy-suggested fixes

### PR DESCRIPTION
**Pull request by rbonichon**

### CODE REVIEW

**Review by mitschabaude — APPROVED** [review]
I added a snarkyjs PR which includes these changes, to test them in CI: https://github.com/o1-labs/snarkyjs/pull/1001

**Comment by dannywillems** [comment]
Let's wait for the CI, otherwise LGTM

**Review by querolita — APPROVED** [review]

**Pull request merged**
